pack.go

```diff
@@ -122,7 +127,7 @@ func PackManifest(ctx context.Context, pusher content.Pusher, packManifestVersio
 	switch packManifestVersion {
 	case PackManifestVersion1_0:
 		return packManifestV1_0(ctx, pusher, artifactType, opts)
-	case PackManifestVersion1_1_RC4:
+	case PackManifestVersion1_1:
 		return packManifestV1_1_RC4(ctx, pusher, artifactType, opts)
```

Comment:
The method name `packManifestV1_1_RC4` should be renamed to `packManifestV1_1`.

Reply:
renamed the method.